src/main/kotlin/com/cognifide/gradle/aem/common/file/downloader/UrlFileDownloader.kt

```diff
@@ -16,9 +17,7 @@ class UrlFileDownloader(val project: Project) {
             val connection = URL(sourceUrl).openConnection()
             connection.useCaches = false
 
-            val downloader = ProgressFileDownloader(project)
-            downloader.size = connection.contentLengthLong
-            downloader.download(connection.getInputStream(), targetFile)
+            IoTransferLogger(project).download(connection.contentLengthLong, connection.getInputStream(), targetFile)
```

Comment:
getter called explicitly? why not using short Kotlin property

Reply:
OK, IntelliJ failed to help with it, but it compiles.